nautobot/extras/filters.py

```diff
@@ -1153,6 +1154,31 @@ class Meta:
         ]
 
 
+class RoleFilter(NaturalKeyOrPKMultipleChoiceFilter):
```

Comment:
Is this going to be reused somewhere else, or why does it need to be a custom class instead of something inline within `RoleModelFilterSetMixin`?

Reply:
on line 1179, I altered the queryset  by applying `.get_for_model(self.model)`, Im not sure there is any-other way to get the `self.model`